test/e2e/cluster_openshift_controller_manager_operator_test.go

```diff
@@ -24,30 +22,26 @@ func TestClusterBuildConfigObservation(t *testing.T) {
 	// make sure the operator is fully up
 	framework.MustEnsureClusterOperatorStatusIsSet(t, client)
 
-	buildConfig := &configv1.Build{
-		ObjectMeta: metav1.ObjectMeta{
-			Name: "cluster",
-		},
-		Spec: configv1.BuildSpec{
-			BuildDefaults: configv1.BuildDefaults{
-				DefaultProxy: &configv1.ProxySpec{
-					HTTPProxy: "testhttpproxy",
-				},
-			},
-		},
+	buildConfig, err := client.Builds().Get("cluster", metav1.GetOptions{})
+	if buildConfig == nil || err != nil {
+		t.Logf("error getting openshift controller manager config: %v", err)
```

Comment:
Simply logging here can lead to a panic - honestly how did this test pass in the first place if the config already exists?

Reply:
updated in latest push